### DISCUSSION TITLE
Own ontogeny file

### DISCUSSION THREAD

**Discussion by njrosen** [Open-Systems-Pharmacology/Forum#1705] [Questions & Problems]
Hi,

I am trying to add my own ontogeny file for some enzymes. My file consists of three columns (PMA in years, ontogeny factor, standard deviation).

When I am uploading the file, it looks all good and PK-sim shows me the correct ranges of the factors in each age. However, once uploaded, the range changes very much. For example, for age 1 year, my mean value is 0.64 and standard deviation 0.125. In the preview it show me range to be 0.51-0.76. However, after uploading the file, the range changes to be 0.08-5.12. 

Am I skipping some step or forgetting something? Any help would be appreciated!

**Comment by Yuri05** [comment]
Can you upload your ontogeny file?
